README.md

```diff
@@ -1,9 +1,13 @@
+![kargo-logo](https://github.com/anevski-stefan/kargo-render/assets/105498279/e6750d98-62aa-4cf2-8164-24b9d26df8a9)
```

Comment:
Two things here:

1. In #186, I said:

    > Layout should be similar to [Kargo's README](https://github.com/akuity/kargo/blob/main/README.md)

   This proved to be wrong on my part. My apologies.

   The logo on the Kargo `README.md` is at the very top of the page and completely replaces the top level heading (single `#`). I forgot that we'd done that. Repeating that layout doesn't make as much sense here. I want the first thing at the top of the page to be clear what repo you are looking at -- Kargo Render.
   
   So the right thing to do here is to leave the top-level heading as it is and _replace_ the octopus logo with the Kargo logo. This does mean that `width` should be set on the Kargo logo so that it isn't enormous.
   
   The octopus logo can be deleted.

1. The graphic shouldn't be served from your fork. Can you please copy it from the Kargo repo and add it to _this_ repo as part of this PR?

I'm very sorry for the confusion.

Reply:
I amended the PR to correct the layout issues.